src/core/engine/processors/postgresql_to_mongo_order_status_history.py

```diff
@@ -0,0 +1,70 @@
+from sqlalchemy import desc
+
+from src.core.engine.processors.abstract_etl_processor import AbstractEtl
+
+from src.lib.entities.sqlalchemy_orm_mapping import (
+    OrderStatusHistory as SqlalchemyOrderStatusHistory,
+)
+
+
+class PostgresqlToMongoOrderStatusHistory(AbstractEtl):
+    def __init__(self, app_processor_config, app_context=None):
+        super().__init__(
+            app_processor_config=app_processor_config, app_context=app_context
+        )
+        self.mongo_connection = None
+        self.postgresql_session = None
+        self.order_status_history_controller = None
+        self.last_postgresql_order_status_id = None
```

Comment:
Same comment here. Maybe you can add a new column.

Reply:
done.